### ISSUE TITLE
CI code workflow started failing

### ISSUE THREAD

**Issue by prjemian** [BCDA-APS/apstools#1017]
About two weeks ago, started receiving consistent failures of the `conde.yml` workflow.
![image](https://github.com/user-attachments/assets/5fbdde48-c4b5-42a5-aa80-bd9f1d652c44)

This is the message
![image](https://github.com/user-attachments/assets/8ebe8e3a-43e0-46d8-9e3e-e8526e9b6bfb)

Definitely blocking the next (1.7.0) release.

**Comment by prjemian** [comment]
From the CI logs:

```
2024-09-06T04:41:41.8039841Z Download action repository 'actions/checkout@v4' (SHA:692973e3d937129bcbf40652eb9f2f61becf3332)
2024-09-06T04:41:41.9262320Z Download action repository 'mamba-org/setup-micromamba@v1' (SHA:f8b8a1e23a26f60a44c853292711bacfd3eac822)
2024-09-06T04:41:42.4187963Z Download action repository 'actions/download-artifact@v4' (SHA:fa0a91b85d4f404e444e00e005971372dc801d16)
```

which has same SHA as https://github.com/actions/download-artifact/releases/tag/v4.1.8

**Comment by prjemian** [comment]
The most recent change in that workflow was 7 months ago.  Still looking for the reason why this started failing now.

**Comment by prjemian** [comment]
Fixed in #1016
